src/composables/useTheme.ts

```diff
@@ -0,0 +1,13 @@
+import { ref } from 'vue';
+
+export function useTheme() {
+  const theme = ref<'light' | 'dark'>('light');
+  const toggleTheme = () => {
+    theme.value = theme.value === 'light' ? 'dark' : 'light';
+  };
```

Comment:
리뷰는 아니지만 아이디어가 생각나서 남겨봅니당.. 
테마 전용의 토글이 아닌 공용 토글로 composable을 생성하여 전역으로 사용하면 좋을 것 같아요!
```suggestion
 
  const toggleState = ref<boolean>(false);
  
  const handleToggle = () => {
     toggleState = !toggleState;
  };
  
  const theme = ref<'light' | 'dark'>('light');
  
  const toggleTheme = () => {
    theme.value = handleToggle() ?  'dark' : 'light';
  };
```